fix(linter): fix whitespace handling in disable directives

lambda47 opened the pull request:
- fixes(#13071)
- Replace ' ' with char::is_whitespace for better whitespace detection

camc314 reviewed (changes requested):
can we add a test for this change please

lambda47 commented:
Sure, sorry, i forget it.

lambda47 committed:
fix(linter): fix whitespace handling in disable directives

- fixes: #13071
- Replace ' ' with char::is_whitespace for better whitespace detection

camc314 reviewed (approved):
thanks!
